apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net/GoogleStackdriverAppender.cs

```diff
@@ -194,6 +201,38 @@ public override void ActivateOptions()
             _isActivated = true;
         }
 
+        private LoggingServiceV2Client BuildLoggingServiceClient()
+        {
+            GoogleCredential credential;
+            if(!TryGetCredentialFromFile(out credential))
+            {
+                return LoggingServiceV2Client.Create();
+            }
+            
+            return LoggingServiceV2Client.Create(new Grpc.Core.Channel(LoggingServiceV2Client.DefaultEndpoint.Host, LoggingServiceV2Client.DefaultEndpoint.Port, Grpc.Auth.GoogleGrpcCredentials.ToChannelCredentials(credential)));
+        }
+
+        private bool TryGetCredentialFromFile(out GoogleCredential credential)

```

Comment:
Please just make this method return `GoogleCredential`, or `null`. We don't use the `Try...(out)` pattern particularly.